db/migrate/20201028191423_create_employees.rb

```diff
@@ -0,0 +1,13 @@
+class CreateEmployees < ActiveRecord::Migration[6.0]
```

Comment:
Is a good practice keeping the migrations in a single commit, and just an example on how we can have smaller commits for making PR Review easier for our teammates.

Otherwise, there are a lot of things we need to take care of in a single commit

Ideally, commits are atomic with a single purpose. While we are attaching only one feature here, it would look like there is only one thing we are doing, but really what we need to do to accomplish this feature can be split into small subtasks.

Just a suggestion for future PR's 👍 

Here I leave a good blog-post about this topic: https://dev.to/cbillowes/why-i-create-atomic-commits-in-git-kfi

Reply:
Got it!